spec/database_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'spec_helper'
+require_relative '../dependencies'
+
+RSpec.describe Database do
+  let(:game_class) { Class.new { extend Database } }
+  let(:path) { 'seed.yaml' }
+
+  let(:summary) do
+    {
+        name: 'Rspec',
+        difficulty: GameStart::DIFFICULTY_LEVEL.keys[0],
+        attempts_total: 15,
+        attempts_used: 1,
+        hints_total: 3,
+        hints_used: 1
+    }
+  end
+
+  describe '#save' do
+    it 'saves a TableData object to a new file' do
```

Comment:
please add contexts:
```
context 'when file does not exist' do
  it 'saves a TableData object to a new file' do
  end
end

context 'when file exists' do
  it 'saves a TableData object to existing file' do
  end
end

```

Reply:
i have done, but i am not sure if what you want me to do.

Reply:
Keep your tests, just add `context`